hudi-utilities/src/test/java/org/apache/hudi/utilities/UtilitiesTestBase.java

```diff
@@ -178,13 +186,40 @@ public static void saveStringsToDFS(String[] lines, FileSystem fs, String target
       os.close();
     }
 
+    public static void saveParquetToDFS(List<GenericRecord> records, Path targetFile) throws IOException {
+      ParquetWriter<GenericRecord> writer = AvroParquetWriter.<GenericRecord>builder(targetFile)
+          .withSchema(HoodieTestDataGenerator.avroSchema).withConf(HoodieTestUtils.getDefaultHadoopConf()).build();
+      for (GenericRecord record : records) {
+        writer.write(record);
+      }
+      writer.close();
```

Comment:
maybe use
```
try(ParquetWriter<GenericRecord> writer = xx) {
    for (GenericRecord record : records) {
        writer.write(record);
    }
} 
```
would be better. since it will close the writer even exception ocurrs while writing record.

Reply:
this is good to know!